CFC_WebApp/main/Trip.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import division
+from dao.user import User
+from googlemaps import GoogleMaps
+from pymongo import MongoClient
+import urllib2
+import xml.etree.ElementTree as ET
+API_KEYS = ("AIzaSyBBcTJ5g9sLrufvAE5BC_KhYZ5ecmbOUxU")
+gmaps = GoogleMaps(API_KEYS[0])
+sectiondb = MongoClient().Stage_database.Stage_Sections
+
+
+def meters_to_miles(meters):
+	return meters * 0.000621371
+
+def calc_car_cost(trip_id, distance):
+	uuid = sectiondb.find_one({'trip_id': trip_id})['user_id']
+	our_user = User.fromUUID(uuid)
+	ave_mpg = our_user.getAvgMpg()
+	gallons =  meters_to_miles(distance) / ave_mpg
+	price = urllib2.urlopen('http://www.fueleconomy.gov/ws/rest/fuelprices')
+	xml = price.read()
+	p = ET.fromstring(xml)[-1]
+	return float(p.text)*gallons
+
+class Trip(object):
```

Comment:
Consider using a common data structure for the stored trips and the alternate trips? Or at least a common base class?
We don't currently have a Trip data access object, but it would be sweet to add one as a cleanup step.